### PR TITLE
Add twig commentstring

### PR DESCRIPTION
**Pull request by gbprod**
Add commentstring for [twig](twig.symfony.com) templating language :)

### CODE REVIEW

**Review by JoosepAlviste — CHANGES_REQUESTED** [review]
Hey, thanks for the PR! I'd change one small thing though

File: lua/ts_context_commentstring/internal.lua [hunk 1]
```diff
@@ -50,6 +50,7 @@ M.config = {
   graphql = '# %s',
   lua = { __default = '-- %s', __multiline = '--[[ %s ]]' },
   vim = '" %s',
+  twig = "{#%s#}",
```
[the remark this answers is not in the document]
Comment (gbprod) [reply lua/ts_context_commentstring/internal.lua]: Resolved !

**Comment by gbprod** [comment]
I think that Stylua Action is broken, I send a fix

**Review by JoosepAlviste — APPROVED** [review]

**Comment by JoosepAlviste** [comment]
Oh seems so, thanks for the contribution!

**Pull request merged**